### DISCUSSION TITLE
Background Images not working

### DISCUSSION THREAD

**Discussion by kyds3k** [wezterm/wezterm#3318] [Q&A] [answered]
I'm trying to recreate the background setup from this page:

https://wezfurlong.org/wezterm/config/lua/config/background.html#parallax-example

But nothing seems to be happening. I've extracted the image files to the Home directory (`c:/users/myname`) and tried starting and restarting, but I don't even get an error. 

Config is at: https://pastebin.com/pvueuEdg

**Comment by wez** [comment, marked as answer]
You need the `File` lines to point to the correct image file path.

Instead of:

```
 File = '/Alien_Ship_bg_vert_images/Backgrounds/spaceship_bg_1.png',
```

you want:

```
 File = 'c:/users/myname/Alien_Ship_bg_vert_images/Backgrounds/spaceship_bg_1.png',
```

**Comment by kyds3k** [comment]
Thank you!
